src/platform/elements/value/Value.ts

```diff
@@ -114,4 +117,11 @@ export class NovoValueElement implements OnInit, OnChanges {
         let isURL: any = Helpers.isString(data) && regex.exec(data.trim());
         return (linkFields.indexOf(field.name) > -1) || !!isURL || field.type === NOVO_VALUE_TYPE.LINK;
     }
+
+    isEntityList(type: string, associatedEntity: { entity: string, entityMetaUrl: string, label: string, fields: any[] }): boolean {
+        if (associatedEntity && associatedEntity.entity === 'Note') {
+            return false;
+        }
+        return type === 'TO_MANY';
+    }
```

Comment:
i think it should be fine to just default to `EntityList` for all to_many OR have the user of `novo-value` set valueType or something to `entity-list` and only display as entity-list if the valueType is set to that value